python/apps/taiga/src/taiga/workflows/api/__init__.py

```diff
@@ -120,3 +108,53 @@ async def create_workflow_status(
         color=form.color,
         workflow=workflow,
     )
+
+
+################################################
+# update workflow status
+################################################
+
+
+@routes.stories.patch(
+    "/projects/{project_id}/workflows/{workflow_slug}/statuses/{status_slug}",
+    name="project.workflow.status.patch",
+    summary="Update workflow status",
+    responses=WORKFLOW_STATUS_200 | ERROR_404 | ERROR_422 | ERROR_403,
+)
+async def update_workflow_status(
+    request: AuthRequest,
+    form: UpdateWorkflowStatusValidator,
+    project_id: B64UUID = Query(None, description="the project id (B64UUID)"),
+    workflow_slug: str = Query(None, description="the workflow slug (str)"),
+    status_slug: str = Query(None, description="the status slug (str)"),
+) -> WorkflowStatusSerializer:
```

Comment:
why can't we use here the simpler way where:
 - we use "response_model"
 - we return "WorkflowStatus"
 - we don't use "WORKFLOW_STATUS_200"

Reply:
That's true, it's simplier. I buy your proposal. :)

Reply:
It also works for events :clap: